src/SDK/MonitoringApplianceEmulator/ViewModels/AlertDetailsControlViewModel.cs

```diff
@@ -143,5 +148,115 @@ private set
         public CommandHandler CloseControlCommand { get; }
 
         #endregion
+
+        private async Task<List<DisplayableAlertProperty>> ComposeArmProperties(AzureResourceManagerRequestAlertProperty armProperty, IAzureResourceManagerClient armClient)
```

Comment:
>ComposeArmProperties [](start = 59, length = 20)

add comment please

Reply:
for all methods

---
In reply to: [316140382](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/72#discussion_r316140382) [](ancestors = 316140382)